### PR TITLE
chore: metrics catalog table container

### PR DESCRIPTION
**Pull request by almeidabbm**
Closes: #12401 

### Description:

image

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/features/metricsCatalog/components/MetricsTable.tsx [hunk 1]
```diff
@@ -119,10 +120,18 @@ export const MetricsTable = () => {
         enableTopToolbar: true,
         mantinePaperProps: {
             shadow: undefined,
+            sx: {
+                border: `1px solid ${theme.colors.gray[2]}`,
+                borderRadius: theme.spacing.sm, // ! radius doesn't have rem(12) -> 0.75rem
```
Comment (almeidabbm) [inline packages/frontend/src/features/metricsCatalog/components/MetricsTable.tsx:125]: To be fixed by: [Fix conflicting styling in figma docs](https://github.com/lightdash/lightdash-internal-work/issues/2215)

File: packages/frontend/src/features/metricsCatalog/components/MetricsTable.tsx [hunk 2]
```diff
@@ -140,29 +149,39 @@ export const MetricsTable = () => {
...
+            <Box
+                p={`${theme.spacing.sm} ${theme.spacing.xl} ${theme.spacing.md} ${theme.spacing.xl}`}
+            >
+                {isFetching ? (
+                    <Text fz={12} fw={500} color="gray.8">
+                        Loading more...
+                    </Text>
+                ) : (
+                    <Group spacing="two">
+                        <Text fz={12} fw={500} color="gray.8">
```
Comment (joaoviana) [inline packages/frontend/src/features/metricsCatalog/components/MetricsTable.tsx:172]:
```suggestion
                        <Text fz='xs' fw={500} color="gray.8">
```

File: packages/frontend/src/features/metricsCatalog/components/MetricsTable.tsx [hunk 3]
```diff
@@ -140,29 +149,39 @@ export const MetricsTable = () => {
...
+                            {hasNextPage
+                                ? 'Scroll for more metrics'
+                                : 'All metrics loaded'}
+                        </Text>
+                        <Text fz={12} fw={400} color="gray.6">
```
Comment (joaoviana) [inline packages/frontend/src/features/metricsCatalog/components/MetricsTable.tsx:177]:
```suggestion
                        <Text fz='xs' fw={400} color="gray.6">
```

**Review by joaoviana — APPROVED** [review]
small NPs

**Pull request merged**